core/trino-main/src/main/java/io/trino/security/AccessControlManager.java

```diff
@@ -1442,6 +1443,18 @@ public Map<ColumnSchema, ViewExpression> getColumnMasks(SecurityContext context,
         }
     }
 
+    @Override
+    public void checkCanSetEntityAuthorization(SecurityContext context, String ownedKind, List<String> name, TrinoPrincipal principal)
```

Comment:
Remove all other `checkCanSet*Authorization` methods from this interface and implementations.

Reply:
Removed.